assets/scripts/modules/GridHelper.js

```diff
@@ -0,0 +1,89 @@
+import { module } from 'modujs';
+
+export default class GridHelper extends module {
+
+    static get settings() {
+        return {
+            GUTTER: 'var(--grid-gutter, 0)',
+            MARGIN: 'var(--grid-margin, 0)',
+            COLOR: 'rgba(255, 0, 0, .1)',
+        }
+    }
+
+    constructor(m) {
+        super(m);
+    }
+
+    init() {
+        this.setColumns()
+        this.setStyles()
+        this.bindEvents()
+
+        window.addEventListener('resize', this.onResize = () => this.setColumns())
+    }
+
+    setColumns() {
+
+        // Clear columns
+        this.el.innerHTML = ''
+
+        // Loop through columns
+        const columns = Number(window.getComputedStyle(this.el).getPropertyValue('--grid-columns'))
+
+        let $col
+        for (var i = 0; i < columns; i++) {
+            $col = document.createElement('div')
+            $col.style.flex = '1 1 0'
+            $col.style.backgroundColor = GridHelper.settings.COLOR
+            this.el.appendChild($col)
+        }
+    }
+
+    setStyles() {
+        const elStyles = this.el.style
+        elStyles.zIndex = '10000'
+        elStyles.position = 'fixed'
+        elStyles.top = '0'
+        elStyles.left = '0'
+        elStyles.display = 'flex'
+        elStyles.width = '100%'
+        elStyles.height = '100%'
+        elStyles.columnGap = GridHelper.settings.GUTTER
+        elStyles.paddingLeft = `calc(${GridHelper.settings.MARGIN} + ${GridHelper.settings.GUTTER})`
+        elStyles.paddingRight = `calc(${GridHelper.settings.MARGIN} + ${GridHelper.settings.GUTTER})`
```

Comment:
I think it should be only `GridHelper.settings.MARGIN` for lateral paddings?

Reply:
Should the GridHelper use grid css instead of flex and padding? It would be coherent with #108

Reply:
The main issue with the grid display is that it doesn't fit with the actual design grid system if we use `fr` units, that's why I used flexbox